### PR TITLE
DAML-SCRIPT: minor cleanup of Struct usage in Converter

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
This PR prepares the ordering of fields in SStruct.

CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [X] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [X] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Converter.scala [hunk 1]
```diff
@@ -209,29 +210,31 @@ object Converter {
       case _ => Left(s"Expected CreateAndExercise but got $v")
     }
 
+  private[this] val tupleFieldNames =
```
Comment (cocreature) [inline daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Converter.scala:213]: Is the idea that this is supposed to be sorted? If so, we should document this in a comment.

Comment (remyhaemmerle-da) [reply daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Converter.scala:213]: It is still not sorted. 
I just want the names and the indexes of the field factored out, so it simplify changes when I make the fields sorted. 
I will change the type of `SStruct#fiedls` in an upcoming PR.

Comment (cocreature) [reply daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Converter.scala:213]: But the indices will be wrong if `tupleFieldNames` is not sorted right?

Comment (remyhaemmerle-da) [reply daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Converter.scala:212]: Currently, the indices should be in the same order as you build the arguments of the `SBStructCon` builtin.
Latter `SBStructCon` will reorder the field, only then the order will be relevant.

I replace the line 
```
 private[this] val Seq(fstIdx, sndIdx) = tupleFieldNames.indices
```
by 
```
  private[this] val fstIdx = 0
  private[this] val sndIdx = 1
``` 
So now, the change really just extract some expressions as values (+ renaming `"a"` to `"fst"` and `"b"` to `"second"`)

Comment (cocreature) [reply daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Converter.scala:212]: Thanks for the explanation!

**Review by cocreature — APPROVED** [review]
Thank you!

**Pull request merged**